packages/eslint-config-airbnb/CHANGELOG.md

```diff
@@ -1,132 +1,172 @@
 6.0.2 / 2016-02-22
 ==================
-- [fix] disable `no-confusing-arrow` due to an `eslint` bug (#752)
+- [fix] disable [`no-confusing-arrow`][no-confusing-arrow] due to an `eslint` bug ([#752](https://github.com/airbnb/javascript/issues/752))
```

Comment:
curious tho, why remove the space before the `-`? In raw form I prefer how it looks with the space.